src/Core/src/Platform/Android/ImageViewExtensions.cs

```diff
@@ -17,6 +17,11 @@ public static void Clear(this ImageView imageView)
 
 		public static void UpdateAspect(this ImageView imageView, IImage image)
 		{
+			if (image.Aspect is Aspect.AspectFill)
```

Comment:
If I delete this code the test still passes

Reply:
A new test has been added, and now the test only passes when the fix is applied. This ensures that the fix is correctly addressing the issue, and without it, the test fails as expected.